doc/vmware_workstation_15_5_0_setup.rst

```diff
@@ -0,0 +1,63 @@
+
+For WMware worktation 15.5.0
+============================
+
+If your installation of VMWare workstation not work with latest version of Windows, you could install old version of WMware workstation.
+
+Here the steps for running `cortx-va-1.0.2.zip </releases/download/VA/cortx-va-1.0.2.zip>`_ for WMware Workstation 15.5.0 in Window 10 version 1890 ( OS build 17763.1637) 
+
+#. Install `WMware Workstation 15.5.0 <https://www.youwindowsworld.com/en/downloads/virtualization/vmware/vmware-workstation-15-pro/download-535-vmware-workstation-15-pro>`_. 
+   (Offial VMWare no longer provided old version for download).
+
+#. After import OVA image to VMware Workstation, go to Menu > Edit > Virtual Network Editor...
+
+   Follow instruction `here <https://github.com/Seagate/cortx/blob/main/doc/troubleshoot_virtual_network.rst>`_  for the IP netowrk setup.
+   
+   If is not working, you need to click on restore default button to restore all network state. 
+   
+   .. image:: images/vmsetup1550/restoreDefault.jpg
+   
+   You might see all IP is same after running **ip a l**, follow instruction below to configure your network.
+
+
+
+If your IP address for all vm is same
+-------------------------------------
+
+Right click image and select settings.
+
+   .. image:: images/vmsetup1550/vmsettings.jpg
+   
+
+#. Click on network adapter, select network connection as briged, check the "replicate pyshical network connection state.
+
+   .. image:: images/vmsetup1550/vm1.jpg
+
+
+#. Select network adapter 2, select Host Only for network connection option.
+
+   .. image:: images/vmsetup1550/vm2.jpg
+
+#. Select network adpater 3, select NAT for network connection option.
+
+   .. image:: images/vmsetup1550/vm3.jpg
```

Comment:
What are the original types of these network interfaces?
You proposed to have:
Bridged,
Host-Only,
NAT.

When the OVA is imported, what are the types of them?
Why are they not working at first place?

Reply:
The network interface follow the one describe in troubleshoot_virtual_network.rst.

The reason I need WMware Workstation on my machine because the latest VMware not work with my WinOS.

And seem like some gap between the 2 WMware workstation release, I follow the instruction in the doc and unfortunately is not working,

I been discuss with Venky  in Slack and he is ok for the new doc. ( we not mentioned any version of OS and the VMware currently, so this try to make thin easier. )

Reply:
Can you please tag the Venky here? I do not remember having this conversation with you.

Regards,
Venky

Reply:
Hi @VenkyOS , I just check back the conversation, seem like I had miss understood, you have mentioned that is a section describe the setup when older version of ESX is used. I can't find any doc related to this.

Not sure If this document will be help for other, since it is for older version of VMware.

Reply:
Please find below the link to the document that I was referring to. This document has the section regarding usage of older version of ESX.

[ESX Server](https://github.com/Seagate/cortx/blob/main/doc/Importing_OVA_File.rst)

Reply:
@VenkyOS I had refer to this document previously when try to run CORTX on older version of VM. 

Since it not mentioned the setup of Virtual Network to work on older version of VM, e.g. 15.5.0,  is that better to put the related info here instead ? (  I mean the page I created in this pull request).